src/table/sticky-columns/__tests__/use-sticky-columns.test.tsx

```diff
@@ -267,7 +265,7 @@ test('cell subscriptions are cleaned up on ref change and un-mount', () => {
   expect(subscribe).toHaveBeenCalledTimes(2);
   expect(unsubscribe).toHaveBeenCalledTimes(1);
 
-  unmount();
+  result.current.ref(null);
```

Comment:
Does this test still work if we use `unmount`? Internally the `unmount` call would then still call `result.current.ref(null)`, but it expresses a bit clearer what we want to simulate in this line than just calling our ref function directly